axelor-human-resource/src/main/resources/views/ProjectPlanningTime.xml

```diff
@@ -114,6 +114,7 @@
 		<context name="_task" expr="eval:__self__" if="_model == 'com.axelor.team.db.TeamTask'"/>
 		<context name="_project" expr="eval:__self__.project" if="_model == 'com.axelor.team.db.TeamTask'"/>
 		<context name="_project" expr="eval:__self__" if="_model == 'com.axelor.apps.project.db.Project'"/>
+		<context name="_timeSpent" expr="eval:false"/>
```

Comment:
Same error is there when we create multiple planning lines from project. So don't change xml but change java code.